src/components/Pressable/PressableWithFeedback.js

```diff
@@ -0,0 +1,52 @@
+import React, {forwardRef} from 'react';
+import _ from 'underscore';
+import propTypes from 'prop-types';
+import GenericPressable from './GenericPressable';
+import GenericPressablePropTypes from './GenericPressable/PropTypes';
+import OpacityView from '../OpacityView';
+import variables from '../../styles/variables';
+import * as StyleUtils from '../../styles/StyleUtils';
+
+const omitedProps = ['style', 'pressStyle', 'hoverStyle', 'focusStyle', 'wrapperStyle'];
```

Comment:
```suggestion
const omittedProps = ['style', 'pressStyle', 'hoverStyle', 'focusStyle', 'wrapperStyle'];
```
For spelling. Should update all references too, of course.